src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -344,14 +344,14 @@ bool
 LedgerMaster::haveLedger (std::uint32_t seq)
 {
     ScopedLockType sl (mCompleteLock);
-    return mCompleteLedgers.hasValue (seq);
+    return boost::icl::contains(mCompleteLedgers, seq);
```

Comment:
Is it worth adding another free function to "hide" the `boost::icl` in this one case? Probably not.